recipes/py2exe/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "py2exe" %}
+{% set version = "0.11.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/py2exe/py2exe/archive/v{{ version }}.tar.gz
+  sha256: 6bb7ff4834ef2096823710c67edf49c971422775fa41c9c7718c19f0437a2d3d
+
+build:
+  number: 0
+  entry_points:
+    - build_exe = py2exe.build_exe:main
+  skip: True  # [ not win ]
+  skip: True  # [ py < 36 ]
+ # for now this is set that way for testing purpose
+  
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - setuptools <60
```

Comment:
Probably worth adding a comment here now that we know this is what is needed. Would also suggest raising an upstream issue about this and linking it here

Reply:
So I basically open an issue to explain the situation with newer setuptools version and also about cachetools and pefile?